client/app/scripts/liveblog-advertising/controllers/advertising.js

```diff
@@ -50,26 +59,33 @@ upload, $templateCache, freetypeService, modal) {
         $scope.advertModalActive = false;
     }
 
-    function loadAdverts() {
-        $scope.advertsLoading = true;
-        api('advertisements').query({not: {term: {deleted: true}}}).then(function(data) {
+    function loadAdverts(silent) {
+        silent = silent || false;
+        !silent ? $scope.advertsLoading = true : $scope.advertsLoading = false;
+        api('advertisements').query({where: {deleted: false}}).then(function(data) {
             $scope.adverts = data._items;
-            notify.info('Adverts loaded');
+            if (!silent) {
+                notify.info('Adverts loaded');
+            }
             $scope.advertsLoading = false;
         }, function(data) {
             $scope.advertsLoading = false;
             notify.error(gettext('There was an error getting the adverts'));
         })
     }
 
-    function handleSaveSuccess() {
+    function handleAdvertSaveSuccess() {
```

Comment:
This is just a mater of how I like the code in a different way. I don't see a better practice.
the only good part is that creating another function isn't needed.
ex: 
`
api('advertisements')
  .save($scope.advert, newAd)
  .then(handleAdvertSaveSuccess)
`